src/lib.rs

```diff
@@ -2239,6 +2222,26 @@ impl Connection {
 
                     ack_eliciting = true;
                     in_flight = true;
+
+                    // if MAX_STREAM_DATA is sent, send MAX_DATA as well.
```

Comment:
```suggestion
                    // Also send MAX_DATA when MAX_STREAM_DATA is sent, to avoid a potential race condition.
```

Reply:
It looks like this change was removed in the latest version, was this on purpose?

Reply:
Oops. Updated. Thanks